runtime/vm/jniinv.c

```diff
@@ -1257,8 +1257,16 @@ jint JNICALL ResetJavaVM(JavaVM *vm)
 /* run the shutdown method in java.lang.Shutdown */
 void 
 sidecarExit(J9VMThread* shutdownThread) {
-	UDATA args[] = {130};	/* The exit code */
-	J9NameAndSignature nas;
+	UDATA args[] = {0};
+	J9NameAndSignature nas = {0};
+
+	/* static void java.lang.Shutdown.exit(int status)
+	 *
+	 * The exit code is 130. Store ints in the low-memory half
```

Comment:
The comment doesn't match the code. If we have 2 slots X & Y (in that order), the original code stored into X for little endian, and Y for big endian. The new code always stores into X. Also instead of AIX I expect you mean big endian platforms. Perhaps args should be declared as I_32 and could continue to be initialized when declared, then cast to UDATA for the call.

Reply:
fixed.